responses/tests/test_recorder.py

```diff
@@ -238,3 +240,145 @@ def _parse_resp_f(file_path):
             assert responses.registered()[3].content_type == "text/plain"
 
         run()
+
+
+class TestRecorderAdditionalHeaders:
+    def setup_method(self):
+        self.out_file = Path("response_record_headers")
+        if self.out_file.exists():
+            self.out_file.unlink()
+        assert not self.out_file.exists()
+
+    def teardown_method(self):
+        if self.out_file.exists():
+            self.out_file.unlink()
+
+    def prepare_server_with_headers(self, httpserver):
+        httpserver.expect_request("/test").respond_with_data(
+            "Test Response",
+            status=200,
+            content_type="text/plain",
+            headers={
+                "Content-Length": "13",
+                "Server": "nginx/1.0",
+                "Connection": "keep-alive",
+                "Content-Encoding": "identity",
+                "Authorization": "Bearer token123",
+                "X-Custom-Header": "custom-value",
+                "User-Agent": "test-agent",
+            },
+        )
+        return httpserver.url_for("/test")
+
+    def test_recorder_with_additional_headers(self, httpserver):
+        url = self.prepare_server_with_headers(httpserver)
+
+        @_recorder.record(
+            file_path=self.out_file,
+            additional_headers=["Authorization", "X-Custom-Header", "Date"],
```

Comment:
```suggestion
            include_headers=["Authorization", "X-Custom-Header", "Date"],
```

I think this reads better, what do you think?